lib/genesis_collector/disks.rb

```diff
@@ -29,9 +29,8 @@ def collect_disks
     private
 
     def disk_healthy(disk)
-      short_tests = shellout_with_timeout("smartctl -H #{disk}", 5)
-
-      short_tests =~ /result: PASSED/
+      shellout_with_timeout("smartctl -H #{disk}", 5)
+      $CHILD_STATUS.success?
```

Comment:
Using a global var set from a called function seems like a bit of an anti-pattern